hudi-client/hudi-spark-client/pom.xml

```diff
@@ -173,6 +183,18 @@
         <artifactId>awaitility</artifactId>
         <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.codehaus.janino</groupId>
+      <artifactId>commons-compiler</artifactId>
+      <version>3.0.10</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.codehaus.janino</groupId>
+      <artifactId>janino</artifactId>
+      <version>3.0.10</version>
```

Comment:
Had to explicitly add these as we were getting NoClassFound for some classes. Will add the version in properties section

Reply:
Has something changed between 2.3.1 to 2.3.9 w.r.t these packages ?

Reply:
After upgrading to hive 2.3.9 many tests failed with 
```
java.lang.NoClassDefFoundError: org/codehaus/janino/InternalCompilerException
```

I will checkout what was different in previous version

Reply:
@n3nash @vinothchandar I am not able to figure this out but the tests fails for sure if janino jar is not present